elfi/methods/parameter_inference.py

```diff
@@ -1132,6 +1132,20 @@ def plot_discrepancy(self, axes=None, **kwargs):
 
         return axes
 
+    def plot_gp(self, axes=None, const=0.5, **kwargs):
+        """Plot pairwise relationships as a matrix with parameters vs. discrepancy.
+
+        Parameters
+        ----------
+        axes : matplotlib.axes.Axes, optional
+
+        Return
+        ------
+        axes : np.array of plt.Axes
+
+        """
+        return vis.plot_gp(self.target_model, self.parameter_names, axes, **kwargs)
+
```

Comment:
It would be good to move all plotting out from this file, for example by using mixin classes.

Can be a separate PR though.